### ISSUE TITLE
mc is slower and shows artifacts

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#1665]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18362.175]   
Windows Terminal version (if applicable): 0.2.1715.0

Screen 1920x1200
default settings for windows terminal
```

# Steps to reproduce
0. open Windows Terminal
1. ssh in any shell session (tested pwsh, cmd, bash) to linux server 
2. start mc (midnight commander)
3. scroll through folder list with arrow down/up

4. do the same as above in pwsh or bash or cmd but started directly (without Windows Terminal)
5. Compare performance and screen quality

# Expected behavior
Performance and stability should be at least the same in both cases

# Actual behavior
Windows Terminal scenario is noticeably slower.
Windows terminal blinks with artifacts (mostly ``|``) on each scroll step when you press and hold up/down key.

Tested the same with mc in local bash session. Less artifacts and a bit improved speed but still noticable slowdown. 

Seems related but not duplicate of #117

**Comment by zadjii-msft** [comment]
For whatever reason, this issue is weirdly bugged and not showing up in the [query I've been using](https://github.com/microsoft/terminal/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Terminal+v1.0%22+no%3Aassignee+-label%3AIn-PR+-label%3AIssue-Docs) to triage [1.0 issues](https://github.com/microsoft/terminal/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Terminal+v1.0%22). 

Thant being said, I'm thinking that this is something that's going to hopefully just magically get fixed with #778/#5185/#5345 getting merged, whenever that does.

**Comment by DHowett-MSFT** [comment]
This was reported in 0.2, actually, before we ironed out some pty issues and some UTF-8 parsing issues that may have resulted in us corrupting things coming out of the client application. I'm going to close this one out, since it doesn't seem to repro any more (at least for me :P) and the speed seems entirely acceptable.

Thanks!
